runtime/vara/src/lib.rs

```diff
@@ -603,7 +601,7 @@ impl pallet_staking::Config for Runtime {
     type ElectionProvider = ElectionProviderMultiPhase;
     type GenesisElectionProvider = onchain::OnChainExecution<OnChainSeqPhragmen>;
     // Burning the reward remainder for now.
-    // TODO: set remainder back to `RewardsStash<Self, Treasury>` to stop burning `Treasury` part.
+    // TODO: set remainder back to `RewardProxy<Self, Treasury>` to stop burning `Treasury` part.
```

Comment:
what this todo is about and when should it be solved?

Reply:
This essentially means that instead of burning tokens minted in excess during the current era (that is, those that have been minted but not supposed to be added to the validator's rewards in order to respect the `max_roi` threshold) we want to move this amount from the offset pool to the `Treasury`. This can only be happening if the current `staking_ratio` is very low (~10% of stakeable tokens and lower) - this will likely never be the case for Vara.

We can switch it back on any time, I guess, but from practical perspective there is no urgency anymore.